static/src/javascripts/projects/common/modules/analytics/coreVitals.ts

```diff
@@ -0,0 +1,133 @@
+import { getCookie } from '@guardian/libs';
+import { getCLS, getFCP, getFID, getLCP, getTTFB } from 'web-vitals';
+
+export const init = () => {
+	coreVitals();
+};
```

Comment:
let's just call the `coreVitals` function `init`, export it, and remove this?

Reply:
Done